### PR TITLE
OSDOCS-9654: Add core namespaces to list of managed resources

### PR DESCRIPTION
**Pull request by bhardesty**
This PR adds the list of core namespaces to the Managed resources doc for OSD and ROSA.

Version(s):
* enterprise-4.15+

Issue:
https://issues.redhat.com/browse/OSDOCS-9654

Link to docs preview:
* OSD: https://77131--ocpdocs-pr.netlify.app/openshift-dedicated/latest/support/troubleshooting/sd-managed-resources#sd-core-namespaces 
* ROSA: https://77131--ocpdocs-pr.netlify.app/openshift-rosa/latest/support/troubleshooting/sd-managed-resources#sd-core-namespaces 

QE review:
- [ ] QE has approved this change.

Additional information:
The managed resources content was originally documented in two identical assemblies: `osd-managed-resources.adoc` and `rosa-managed-resources.adoc`. Since these files contain exactly the same content, I changed it to a single `sd-managed-resources.adoc` assembly and reused it in the OSD and ROSA docs.

### CODE REVIEW

**Comment by bhardesty** [comment]
@laubai I added the list of core namespaces to the OSD and ROSA docs. Let me know your thoughts!

File: support/troubleshooting/sd-managed-resources.adoc [hunk 1]
```diff
@@ -1,17 +1,17 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="rosa-managed-resources"]
+[id="osd-managed-resources"]
```
Comment (adellape) [inline support/troubleshooting/sd-managed-resources.adoc:2]:
```suggestion
[id="sd-managed-resources"]
```
?

**Comment by xueli181114** [comment]
The link of the doc preview cannot be accessed

**Comment by bhardesty** [comment]
@xueli181114 Can you access the preview now?

OSD: https://77131--ocpdocs-pr.netlify.app/openshift-dedicated/latest/support/troubleshooting/sd-managed-resources#sd-core-namespaces

ROSA: https://77131--ocpdocs-pr.netlify.app/openshift-rosa/latest/support/troubleshooting/sd-managed-resources#sd-core-namespaces

**Comment by xueli181114** [comment]
LGTM

File: support/troubleshooting/sd-managed-resources.adoc [hunk 2]
```diff
@@ -39,7 +53,7 @@ include::https://raw.githubusercontent.com/openshift/managed-cluster-config/mast
 ----
 ====
 
-[id="rosa-validating-webhooks"]
+[id="sd-validating-webhooks"]
```
Comment (ShaunaDiaz) [inline support/troubleshooting/sd-managed-resources.adoc:56]:
```suggestion
[id="sd-validating-webhooks_{context}"]
```
Can't hurt to add the context tags?

File: support/troubleshooting/sd-managed-resources.adoc [hunk 3]
```diff
@@ -25,7 +25,21 @@ include::https://raw.githubusercontent.com/openshift/managed-cluster-config/mast
...
+.List of core namespaces
+[%collapsible]
+====
+[source,yaml]
+----
+include::https://raw.githubusercontent.com/openshift/managed-cluster-config/master/deploy/osd-managed-resources/ocp-namespaces.ConfigMap.yaml[]
+----
+====
+
+[id="sd-add-on-managed-namespaces"]
```
Comment (ShaunaDiaz) [inline support/troubleshooting/sd-managed-resources.adoc:42]:
```suggestion
[id="sd-add-on-managed-namespaces_{context}"]
```

File: support/troubleshooting/sd-managed-resources.adoc [hunk 4]
```diff
@@ -25,7 +25,21 @@ include::https://raw.githubusercontent.com/openshift/managed-cluster-config/mast
 ----
 ====
 
-[id="rosa-add-on-managed-namespaces"]
+[id="sd-core-namespaces"]
```
Comment (ShaunaDiaz) [inline support/troubleshooting/sd-managed-resources.adoc:28]:
```suggestion
[id="sd-core-namespaces_{context}"]
```

File: support/troubleshooting/sd-managed-resources.adoc [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
 toc::[]
 
-[id="rosa-managed-resources-overview"]
+[id="sd-managed-resources-overview"]
 == Overview
 
 The following covers all resources managed or protected by the Service Reliability Engineering Platform (SRE-P) Team.  Customers should not attempt to modify these resources because doing so can lead to cluster instability.
 
-[id="rosa-managed-resources-all"]
+[id="sd-managed-resources-all"]
```
Comment (ShaunaDiaz) [inline support/troubleshooting/sd-managed-resources.adoc:14]:
```suggestion
[id="sd-managed-resources-all_{context}"]
```

File: support/troubleshooting/sd-managed-resources.adoc [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
+[id="sd-managed-resources"]
 = {product-title} managed resources
 include::_attributes/attributes-openshift-dedicated.adoc[]
-:context: rosa-managed-resources
+:context: sd-managed-resources
 
 toc::[]
 
-[id="rosa-managed-resources-overview"]
+[id="sd-managed-resources-overview"]
```
Comment (ShaunaDiaz) [inline support/troubleshooting/sd-managed-resources.adoc:9]:
```suggestion
[id="sd-managed-resources-overview_{context}"]
```

**Comment by ShaunaDiaz** [comment]
@bhardesty It looks fine, but it might be nice to add the context tags.

**Pull request merged**